src/index.html

```diff
@@ -1,429 +1,398 @@
 <!DOCTYPE html>
 <html lang=nl>
 <head>
-    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css'/>
-    <title>DataPunt APIs en Geoservices</title>
+<link rel="stylesheet" href="https://fast.fonts.net/cssapi/3680cf49-2b05-4b8a-af28-fa9e27d2bed0.css">
+<title>DataPunt APIs en Geoservices</title>
+<style>
+body, input, textarea, keygen, select, button {
+	font-size: 16px;
+	line-height: 1.25;
+	color: #000;
+	font-family: "Avenir LT W01 55 Roman", Arial, sans-serif;
+}
+html, body {
+	height: 100%;
+	margin: 0;
+	padding: 0;
+}
+.header {
+	display: block;
+	height: 50px;
+	border-bottom: 2px #999 solid;
+}
+.header_logo {
+	position: absolute;
+	top: 10px;
+	left: 10px;
+}
+.header_titel {
+	line-height: 50px;
+	color: #000;
+	margin-left: 85px;
+	font-weight:bold;
+}
+.header a {
+	text-decoration:none;
+	color: #000;
+}
+.container {
+	padding: 0px;
+}
+.container a {
+	color: #000;
+}
+h1 {
+	margin: 20px 0 10px 20px;
+	line-height: 26px;
+	font-size: 26px;
+}
+h2 {
+	margin: 0 0 30px 20px;	
+	line-height: 18px;
+	font-size: 18px;
+}
+thead {
+	height: 26px;
+	background-color: #999999;	
+}
+table {
+	padding: 0;	
+	width: 100%;
+	margin-bottom: 40px;
+}
+thead tr {
+	color: #fff;
+	text-align:left;
+}
+th, td {
+	padding-left: 20px;	
+}
+.column1 {
+	width: 30%;
+}
+.column2 {
+	width: 10%;
+}
+.column3 {
+	width: 19%;
+}
+.column4 {
+	width: 30%;
+}
+.column5 {
+	width: 11%;
+}
+tbody tr, tbody td {
+	border-bottom: #ccc 1px solid;	
+	line-height: 20px;
+	padding: 10px 0 10px 20px;
+}
+tbody img {
+	height: 26px;	
+}
+</style>
 </head>
 <body>
-
+<div class="header"> <a href="https://atlas.amsterdam.nl/#?mpb=topografie&mpz=9&mpv=52.3719:4.9012&pgn=home" title="Naar voorpagina Dataportaal"> <img class="header_logo" src="logo-short.svg" width="64" height="30"> <span class="header_titel">data.amsterdam.nl</span> </a> </div>
 <div class='container'>
     <h1>REST APIs </h1>
-    <h4>Source code: <a href="https://github.com/DatapuntAmsterdam">github.com/DatapuntAmsterdam</a></h4>
-    <table class="table">
+    <h2>Source code: <a href="https://github.com/DatapuntAmsterdam" target="_blank">github.com/DatapuntAmsterdam</a></h4>
```

Comment:
Target _blank is nogal gebruiksonvriendelijk omdat je de keuze uit de handen van de gebruiker neemt.